lighthouse-cli/test/smokehouse/smokehouse-report.js

```diff
@@ -199,22 +226,22 @@ function reportAssertion(assertion) {
 /**
  * Log all the comparisons between actual and expected test results, then print
  * summary. Returns count of passed and failed tests.
- * @param {Smokehouse.LHRComparison} results
+ * @param {Smokehouse.Comparison[]} comparisons
  * @return {{passed: number, failed: number}}
  */
-function report(results) {
+function report(comparisons) {
   let correctCount = 0;
   let failedCount = 0;
 
-  [results.finalUrl, results.errorCode, ...results.audits].forEach(auditAssertion => {
-    if (auditAssertion.equal) {
+  comparisons.forEach(assertion => {
+    if (assertion.equal) {
```

Comment:
feels like we've still got quite a bit of assertion/comparison mixing going on throughout but that was preexisting :)

Reply:
It feels like it should always be comparison? Assertion seems more narrow in scope to me than what we have here.

Reply:
agreed :) cleanup for another time